features/place/description.feature

```diff
@@ -49,3 +49,25 @@ Feature: Test place description property
     """
     ""
     """
+
+  Scenario: Delete the last description of a place
+    When I send a DELETE request to "%{placeUrl}/description/nl"
+    Then the response status should be "204"
+    And I send a GET request to "%{placeUrl}"
+    Then the response status should be "200"
+    And the JSON response should not have "description"
```

Comment:
Same issue here, the used place has no description to start with

Reply:
I now create the descriptions with a put request